sourcegraph/sourcegraph
# desktop 404 listener

How to test:

- Run Sourcegraph Desktop (ensure pointed at this local server)
add view.openDevTools() to `dom-ready` in `DefInfoView.js` in app
- Trigger error (click symbol on unindexed repo)
- Ensure error is sent in `EventLogger.js` (sourcegraph/sourcegraph)


cc @KattMingMing @uforic
